easybuild/easyconfigs/p/picard/picard-2.10.1-Java-1.8.0_131.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+# License::	MIT 
+#
+# 2.10.1:
+# Adam Huffman
+# The Francis Crick Institute
+#
+##
+
+easyblock = 'JAR'
+
+name = 'picard'
+version = '2.10.1'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://sourceforge.net/projects/picard'
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/broadinstitute/picard/releases/download/%(version)s']
+sources = ['%(name)s.jar']
```

Comment:
Use the rename-after-download support to properly version this?

You'll also need to include `postinstallcmds` to rename the install file to `picard.jar`?